scripts/templates/print.hpp.mako

```diff
@@ -381,14 +389,14 @@ inline std::ostream &operator<<(std::ostream &os, [[maybe_unused]] const struct
 %endfor
 %endfor
 
-namespace ${x}_params {
-
-template <typename T> inline void serializePtr(std::ostream &os, const T *ptr) {
+///////////////////////////////////////////////////////////////////////////////
+// @brief Print pointer value
+template <typename T> inline ${x}_result_t ${x}PtrPrint(std::ostream &os, const T *ptr) {
```

Comment:
This should not be part of the public API. Maybe we should have a `ur_print::details` namespace?